app/_layouts/base.njk

```diff
@@ -1,16 +1,15 @@
 {% extends "govuk/template.njk" %}
 
-{% set htmlClasses = 'no-js' %}
+{% set htmlClasses = "no-js" %}
 
 {% from "header/macro.njk" import appHeader %}
 {% from "breadcrumbs/macro.njk" import appBreadcrumbs %}
 
 {% block head %}
-<!--[if lte IE 8]><link href="/stylesheets/application-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
-<!--[if gt IE 8]><!--><link href="/stylesheets/application.css" media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
+<!--[if lte IE 8]><link href="/stylesheets/application-ie8.css" rel="stylesheet" type="text/css"><![endif]-->
+<!--[if gt IE 8]><!--><link href="/stylesheets/application.css" media="all" rel="stylesheet" type="text/css"><!--<![endif]-->
```

Comment:
Can we remove the IE8 stuff altogether?

Reply:
Lets. I’ll remove it in one of the other closely related PRs.

Reply:
Scratch that, I already did in #71.